### PR TITLE
:lipstick: {refactor} Side nav layout #24

### PR DESCRIPTION
**Pull request by faisoabdirisak**
# Description

I have edited and designed the side nav according to the design needed.

Reference [here](https://github.com/italanta/italanta-apps/issues).

Fixes # (issue) -

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# Screenshot (optional)

Screenshot (504)

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: libs/elements/layout/page-convl/src/lib/components/convl-sidemenu/convl-sidemenu.component.html [hunk 1]
```diff
@@ -14,31 +14,36 @@
         <div [routerLink]="['/']" mat-list-item class="header top-header" fxLayout="row" fxLayoutAlign="start center">
 
             <!-- <i matListIcon class="fas fa-columns"></i> -->
-            <img class="list-icon icon-top" src="assets/images/ele-small-ico.png">
-            <div fxLayout="column">
+            <img class="list-icon" src="assets/images/elewa-logo.png">
+            <div class="menu">
+              <i class="fas fa-bars"></i>
```
Comment (ReaganCn) [inline libs/elements/layout/page-convl/src/lib/components/convl-sidemenu/convl-sidemenu.component.html:19]: Use fort awesome icons for consistency across the codebase

File: libs/elements/layout/page-convl/src/lib/components/convl-sidemenu/convl-sidemenu.component.html [hunk 2]
```diff
@@ -14,31 +14,36 @@
...
-                    <i class="fas fa-sort-down"></i>
+                    <!-- <i class="fas fa-sort-down"></i> -->
                 </div>
                 <ng-template #icon2><i class="fas fa-sort-down chevron-down"></i></ng-template>
             </div>
 
             <div class="category-children" *ngIf="rlaHome.isActive" fxLayout="column" fxLayoutGap="5px">
                 <div [routerLink]="['/org']" mat-list-item class="child-header child-1">
-                    {{'PAGE-CONTENT.SIDE-MENU.DASHBOARD'| transloco }}
+                  <i class='fas fa-tachometer-alt'></i>
```
Comment (ReaganCn) [inline libs/elements/layout/page-convl/src/lib/components/convl-sidemenu/convl-sidemenu.component.html:45]: Same here. Use fort awesome

File: libs/features/convs-mgr/home/src/lib/components/story-list/story-list.component.html [hunk 3]
```diff
@@ -1,18 +1,13 @@
 
-
 <!-- edited the search -->
   <div class="search">
    
-    <fa-icon [icon]="searchIcon" class="fa" ></fa-icon>
+    <i class="fa fa-search"></i>
```
Comment (ReaganCn) [inline libs/features/convs-mgr/home/src/lib/components/story-list/story-list.component.html:5]: Same here. Use fort awesome

**Review by ReaganCn — CHANGES_REQUESTED** [review]
Good approach. See additional comments below

File: libs/elements/layout/page-convl/src/lib/components/convl-sidemenu/convl-sidemenu.component.html [hunk 4]
```diff
@@ -14,31 +14,36 @@
...
         <hr id="menu-header-seperator" />
 
         <!-- Section Home -->
         <div [routerLink]="['/']" routerLinkActive #rlaHome="routerLinkActive" mat-list-item class="header" [class.active]="rlaHome.isActive" [class.hov]="!rlaHome.isActive" fxLayout="column">
 
-            <div fxLayout="row">
-                <i matListIcon class="fas fa-home list-icon"></i> Home
+            <div fxLayout="row"  fxLayoutAlign="start" class="sub-header">
+                <!-- <i matListIcon class="fas fa-home list-icon"></i> Home -->
+                <span class="main-text">Main</span>
```
Comment (ReaganCn) [inline libs/elements/layout/page-convl/src/lib/components/convl-sidemenu/convl-sidemenu.component.html:36]: Use multi-lang

**Comment by faisoabdirisak** [comment]
made  the changes

**Review by ReaganCn — APPROVED** [review]
Good work! Keep it up

**Pull request merged**